backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -633,7 +633,7 @@ public void deleteBranchByCommit(Long repoId, String commitHash) {
       while (!childCommitSHAs.isEmpty()) {
         String childCommit = childCommitSHAs.remove(0);
         commitSHAs.add(childCommit);
-        Query sqlQuery = session.createSQLQuery(getParentCommitsQuery + childCommit + "\'");
+        Query sqlQuery = session.createSQLQuery(getParentCommitsQuery + childCommit + "\"");
```

Comment:
single quote for Postgres . Also slightly surprized that doing in just here and not the prefix worked.

Reply:
removed the change